### PR TITLE
sets model to eval mode (required due to batchnorm layer)

### PR DESCRIPTION
**Pull request by ranahanocka**
the current training scripts do not set the (classifier) model to eval during test, which means that the batch norm layer is not loading the (learned) parameters for inference.

### CODE REVIEW

**Comment by fxia22** [comment]
thanks!
